README.md

```diff
@@ -96,61 +93,67 @@ our [official documentation](https://docs.conjur.org/Latest/en/Content/Developer
 
 To start using the SDK in your applications, create a Client instance and then invoke the API on it:
 
-#### With login ID and password
+#### Define connection parameters
```

Comment:
is this step 1 of the process of creating a client instance and invoking the API?

Reply:
yep